internal/pkg/template/workload.go

```diff
@@ -275,6 +264,11 @@ type importableValue interface {
 	Value() string
 }
 
+// IsHTTPS returns true if the target container's port is 443.
+func (lr ALBListenerRule) IsHTTPS() bool {
```

Comment:
This function signature says "whether the listener rule is an HTTPS rule", which does not seem to be accurate. What do you think of either of the following:
```go
type TargetPort string
func (p TargetPort) IsHTTPS() { return string(p) == "443" }
```
or
```go
	switch {
	case lr.HTTPHealthCheck.Port == "443":
		return "HTTPS"
	case lr.TargetPort == "443" && lr.HTTPHealthCheck.Port == "":
		return "HTTPS"
	case lr.TargetPort == "443" && lr.HTTPHealthCheck.Port != "443":
```